crates/bevy_render/src/render_phase/draw.rs

```diff
@@ -15,20 +15,23 @@ use bevy_ecs::{
 };
 use bevy_utils::HashMap;
 use parking_lot::{RwLock, RwLockReadGuard, RwLockWriteGuard};
-use std::{any::TypeId, fmt::Debug, hash::Hash, ops::Range};
+use std::{any::TypeId, fmt::Debug, hash::Hash};
 
-/// A draw function which is used to draw a specific [`PhaseItem`].
+// Todo: consider renaming this to `DrawFunction`
```

Comment:
Move this into an issue please <3 Much easier to track and discuss.

Reply:
Done: #7158